basics/installation/install-from-cli.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Installing PrestaShop from CLI
+menuTitle: Installation from CLI
+weight: 15
+---
+
+# Installing PrestaShop from CLI
+
+Since version 1.5.4, PrestaShop has a command-line installer.
+
+This special installer makes it possible to install PrestaShop without the need to use a web browser: simply put the content of the zip archive on your web server or pull code from an official PrestaShop repository, and you can install PrestaShop through your command-line interface (CLI). 
+
+{{% notice info %}}
+If deploying from sources (PrestaShop repository), you must first install Composer dependencies.
+Install them with `composer install` from project's root directory.
+{{% /notice %}}
+
+The point of having a CLI installer in addition to the regular in-browser installer is to give this option to cater for some advanced users, who often prefer command-line interfaces as they tend to provide a more concise and powerful means to control a program or operating system.
+
+## How to use it
+
+The CLI installer is easy to use: from your terminal, go to the /install (or /install-dev) folder, and start the script with this command:
+
+```shell
+php index_cli.php
+```
+
+This will display the various available options:
+
+```
+Arguments available:
+--step	all / database,fixtures,theme,modules,postInstall	(Default: all)
+--language	language iso code	(Default: en)
```

Comment:
Worth describing each option. E.g. firstname, name of the administrator, email: used to login to the back office, modules: install all by default or provided as a parameter, etc.